products/firefox_android/localize_android.md

```diff
@@ -0,0 +1,45 @@
+# Firefox for Android - How to localize
+
+Please make sure to go through the [Firefox for Android README](/products/firefox_android/README.md) first to get a clear overview of what this project covers.
```

Comment:
I have the feeling you misread my comment (or it wasn't clear): I was asking you to add a link to these 2 documents in the README, not to link the README from these docs.